

process.on('uncaughtException', (err)=>{
    logger.error(err);
})

